packages/json5/json5.mjs

```diff
@@ -0,0 +1,11 @@
+import JSON5 from 'json5';
+
+export function loadJSON5(source) {
+	try {
+		const parsed = JSON5.parse(source);
+
+		return parsed;
+	} catch (error) {
+		throw new Error(`JSON5 parsing error: ${error.message}`);
+	}
```

Comment:
What does the error JSON5 itself throws look like? I'm wondering if this try/catch is necessary.

Reply:
I agree with what you want to say here, json5 already has an error mechanism in itself, it would be good to leave this place simple, in addition, I added a test for the error mechanism of json5.